conda_forge_tick/make_graph.py

```diff
@@ -26,7 +27,8 @@
 
 
 NUM_GITHUB_THREADS = 4
-
+BATCH_SIZE = 10
+SLEEP_TIME = 60
```

Comment:
Maybe this is too large a unit of time. https://developer.github.com/v3/rate_limit/

Reply:
I'd probably go for some light exponential back off here capping at 60s

Reply:
Odds are we should just put this block into something that we just allocate a fixed maximal time for, eg if we've been updating for 10 minutes just stop.